priv/big_query/migrations/20220407195034_create_podcast_metadata_tables.exs

```diff
@@ -0,0 +1,46 @@
+defmodule BigQuery.Migrations.CreatePodcastMetadataTables do
+  alias BigQuery.Base.Query
+
+  def up do
+    Query.log("""
+      CREATE TABLE podcasts
+      (
+        id INT64 NOT NULL,
```

Comment:
A bit odd - but the foreign key for this in `dt_downloads` and `dt_impressions` is called `feeder_podcast`.  So it doesn't exactly follow the normal naming standards.  But I don't know that I want to rename that column, as it's been around for so long.

I think we can just iron this stuff out in the export-SQL?